src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPCTest.java

```diff
@@ -84,8 +88,9 @@ private static DataModel createModel() {
 			MAX_TX_POWER, BSSID_A, 36, DEFAULT_CHANNEL_WIDTH, MAX_TX_POWER, BSSID_A);
 		State stateB = TestUtils.createState(1, DEFAULT_CHANNEL_WIDTH,
 			MAX_TX_POWER, BSSID_B, 36, DEFAULT_CHANNEL_WIDTH, MAX_TX_POWER, BSSID_B);
-		State stateC = TestUtils.createState(1, DEFAULT_CHANNEL_WIDTH,
-			MAX_TX_POWER, BSSID_C, 36, DEFAULT_CHANNEL_WIDTH, MAX_TX_POWER, BSSID_C);
+		State stateC = TestUtils.createState(
+			1, DEFAULT_CHANNEL_WIDTH, MAX_TX_POWER, BSSID_C
+		);
```

Comment:
Is there a reason why `stateC` is created differently from the others?

Reply:
Good catch. Actually, this needs some more cleaning up, which I'll do after vacation :P For now, I'm putting stateC back to how it was.